lib/services/systemd.c

```diff
@@ -321,8 +321,9 @@ set_result_from_method_error(svc_action_t *op, const DBusError *error)
                                "systemd unit %s not found", op->agent);
     }
 
-    crm_info("DBus request for %s of systemd unit %s for resource %s failed: %s",
+    crm_info("DBus request for %s of systemd unit %s%s%s failed: %s",
              op->action, op->agent, pcmk__s(op->rsc, "(unspecified)"),
```

Comment:
pcmk__s(op->rsc, "(unspecified)") should probably go away.
interesting that gcc didn't stumble over it.

Reply:
Whoops, that is odd